project-ui/Dockerfile

```diff
@@ -1,6 +1,12 @@
-FROM node:15.13-alpine
+FROM balenalib/raspberrypi4-64-alpine-node
```

Comment:
Unless you need a different image for some specific reason, I usually tend to try and use balena images when working on a balena project. Non-balena images are fine, but I'd rather dogfood when possible.